apps/src/templates/projects/projectsModule.js

```diff
@@ -1,10 +1,16 @@
 /** @file Redux actions and reducer for the Projects Gallery */
 import {registerReducers} from '@cdo/apps/redux';
+import _ from 'lodash';
 
 // Action types
+
 const TOGGLE_GALLERY = 'projectsModule/TOGGLE_GALLERY';
+const ADD_OLDER_PROJECTS = 'projectsModule/ADD_OLDER_PROJECTS';
+const SET_PROJECT_LISTS = 'projectsModule/SET_PROJECT_LISTS';
+const SET_HAS_OLDER_PROJECTS = 'projectsModule/SET_HAS_OLDER_PROJECTS';
```

Comment:
Gamelab does appear to be setup slightly differently. The pattern I've been targeting in general is as follows:
- Our redux state tree is split into multiple subtrees (i.e. stageLock, pageConstants, etc.)
- Each of those subtrees should have its own "duck module" (https://github.com/erikras/ducks-modular-redux).
- These modules should have action names where the prefix matches the subtree name (i.e. `stageLock/OPEN_LOCK_DIALOG'). They should also have action creators, and a reducer that handles all the different actions.
- Outside of your modules there will be some code that is setting up your app. It should import the reducers from each of your duck modules, and then call registerReducers (our self-configured way or informing our singular redux store that these are reducers we care about)